### PR TITLE
Improvement: Horizontal scrollbar styling

### PR DESCRIPTION
**Pull request by alexis-ld**
**Component**: ScrollbarWrapper

**Description**: ScrollbarWrapper was missing rules for horizontal scrollbar styling.

**Breaking Changes**: none

---

### CODE REVIEW

File: src/lib/components/scrollbarwrapper/ScrollbarWrapper.component.js [hunk 1]
```diff
@@ -25,12 +26,15 @@ const ScrollbarContainer = styled.div`
...
           width: 4px;
+          height: 4px;
           background: ${brand.border};
           border-radius: 4px;
           -webkit-border-radius: 4px;
           background-clip: padding-box;
           border-left: 2px solid rgba(0, 0, 0, 0);
           border-right: 2px solid rgba(0, 0, 0, 0);
+          border-top: 2px solid rgba(0, 0, 0, 0);
+          border-bottom: 2px solid rgba(0, 0, 0, 0);
```
Comment (ChengYanJin) [inline src/lib/components/scrollbarwrapper/ScrollbarWrapper.component.js]: Would be better to merge the four lines into one line.

**Review by ChengYanJin — CHANGES_REQUESTED** [review]

File: src/lib/components/scrollbarwrapper/ScrollbarWrapper.component.js (hunk 1 above)
Comment (alexis-ld) [reply src/lib/components/scrollbarwrapper/ScrollbarWrapper.component.js]: Absolutely ^^ sorry about that. Done

**Review by ChengYanJin — APPROVED** [review]
LGTM!

**Pull request merged**